Spinne and music added

d-schn opened the pull request:
Added Spinne NPC west of the player with animation and music player on getting closer
known issues: music doesn't play in full, playback start etc. not fully tested yet

PR so changes to main files can be updated in main branch.

d-schn committed:
Spinne and music added

Added Spinne NPC west of the player with animation and music player on getting closer
known issues: music doesn't play in full, playback start etc. not fully tested yet

d-schn committed:
radius adjustment

Adjusted music radius

g-oelk reviewed (approved):
Tested playback distance, 600 seems a little low better 800 for the extra thrill of not seeing but hearing. Music should get significantly louder when spinne is in sight. Otherwise nice.

d-schn committed:
Adjusted radius, music boost when on screen

Adjusted music radius to 800 as suggested, added 5dB boost when Spinne is in sight
